### PR TITLE
DXCDT-314: Add ability to remove roles from a user

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

Add ability to remove roles from a user.

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/auth0/user.go [hunk 1]
```diff
@@ -33,4 +33,7 @@ type UserAPI interface {
 
 	// AssignRoles assigns roles to a user.
 	AssignRoles(id string, roles []*management.Role, opts ...management.RequestOption) error
+
+	// RemoveRoles removes any roles associated to a user.
```
Comment (Widcket) [inline internal/auth0/user.go:37]:
```suggestion
	// RemoveRoles removes roles from a user.
```

**Review by willvedd — COMMENTED** [review]
Given that this is a new command, can we add some type of commander test?

File: internal/cli/users_roles.go [hunk 2]
```diff
@@ -141,7 +142,67 @@ func addUserRolesCmd(cli *cli) *cobra.Command {
...
+
+			var rolesToRemove []*management.Role
+			for _, roleID := range inputs.Roles {
+				rolesToRemove = append(rolesToRemove, &management.Role{
+					ID: auth0.String(roleID),
+				})
+			}
+
+			if err := ansi.Waiting(func() (err error) {
+				return cli.api.User.RemoveRoles(inputs.ID, rolesToRemove)
```
Comment (willvedd) [inline internal/cli/users_roles.go:180]: Should check to see if length of `rolesToRemove` is non-zero, otherwise will result in a 400.

**Review by willvedd — CHANGES_REQUESTED** [review]

**Review by willvedd — APPROVED** [review]
LGTM

**Pull request merged**